File: server/aoNewsletter2/Controllers/RssFeedController.vb
Imports System.Linq.Expressions
Imports Contensive.BaseClasses

Public Class RssFeedController
    '
    Public Shared Sub updateRSSFeed(cp As CPBaseClass)
        '
        cp.Db.ExecuteNonQuery($"upadte ccaggregatefunctions set processRunOnce=1 where name='RSS Feed Process'")
        '
    End Sub
End Class

File: server/aoNewsletter2/Controllers/RssFeedController.cs
using Contensive.BaseClasses;

namespace Contensive.Addons.Newsletter {

    public class RssFeedController {
        // 
        public static void updateRSSFeed(CPBaseClass cp) {
            // 
            cp.Db.ExecuteNonQuery($"update ccaggregatefunctions set processRunOnce=1 where name='RSS Feed Process'");
            // 
        }
    }
}